recipe/meta.yaml

```diff
@@ -19,19 +19,20 @@ build:
 
 requirements:
   build:
-    - toolchain
-    - m2w64-toolchain  # [win and py2k]
+    - {{ compiler('c') }}        # [not (win and py2k)]
+    - {{ compiler('m2w64_c') }}  # [win and py2k]
```

Comment:
Would think selectors on `compiler`s would be respected when re-rendering, but that doesn't seem to be the case. Appears to be a known issue ( https://github.com/conda/conda-build/issues/2958 ).